src/pages/about.astro

```diff
@@ -4,9 +4,38 @@ import Content from "../components/Content.astro";
 ---
 
 <Layout title="About this project">
-  <Content title="What is this project about" />
-  <div>
-    This is my first Astro project where I found my first aproach to a framework
-    use.
+  <Content title="What is this project about?" />
+  <div class="px-4 md:px-6 lg:px-8 xl:px-10">
+    <h2 class="font-bold text-2xl px-6 text-lila">Trying my skills</h2>
+  </div>
+  <div class="px-32 py-6 leading-10">
+    <p class="leading-7">
```

Comment:
It's very good! but, maybe you can put all content in a json file and pick and loop them